prefect_dask/task_runners.py

```diff
@@ -256,7 +266,11 @@ async def _start(self, exit_stack: AsyncExitStack):
         - Creates a client to connect to the cluster.
         - Pushes a call to wait for all running futures to complete on exit.
         """
-        if self.address:
+
+        if self._cluster:
+            self.logger.info(f"Connecting to existing Dask cluster {self._cluster}")
+            connect_to = self._cluster
```

Comment:
It seems like we could still support `adapt_kwargs` in this case.

Reply:
Sure, that feels like an additional enhancement beyond what I was trying to accomplish here, but I can do that too.